breathe/renderer/sphinxrenderer.py

```diff
@@ -351,11 +351,22 @@ def set_context(self, context: RenderContext) -> None:
         if self.context.domain == '':
             self.context.domain = self.get_domain()
 
+    def _strip_doxygen_anchorid(self, refid: str) -> str:
```

Comment:
Could you put a comment above this function explaining why this is needed? Copy/pasting some of your PR comments should be good. Just to help out future maintenance.

Perhaps annotate with `XXX` or similar too.

Thanks for the patch, rest seems good to me. (I will handle most other PRs and make a release tomorrow, for those wondering.)

Reply:
Done!

Reply:
> Thanks for the patch, rest seems good to me. (I will handle most other PRs and make a release tomorrow, for those wondering.)

Put this PR on hold for now, while looking at the Doxygen's source code it seems that this only works as a band-aid if whatever ref is being pointed to, exists inside a group (and added with a `.. doxygengroup:` directive). I will submit a patch to Doxygen and based on feedback I will rework how to "band-aid" it here.